pkg/injector/injector.go

```diff
@@ -179,6 +181,13 @@ func (i *injector) handleRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var pod corev1.Pod
+	if err := json.Unmarshal(ar.Request.Object.Raw, &pod); err != nil {
```

Comment:
can we do just type conversion ? - ar.Request.Object.(*corev1.Pod)

Reply:
That failed for me before